File: eventos-api/app/models/inscricao.py
from sqlalchemy import Column, DateTime, String, ForeignKey, Boolean
from sqlalchemy.dialects.postgresql import UUID
import uuid, datetime
from app.core.database import Base

class Inscricao(Base):
    __tablename__ = "inscricoes"
    id = Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
    evento_id = Column(UUID(as_uuid=True), ForeignKey("eventos.id"), nullable=False)
    usuario_id = Column(UUID(as_uuid=True), ForeignKey("usuarios.id"), nullable=True)
    inscricao_rapida = Column(Boolean)
    nome_rapido = Column(String(200), nullable=False)
    cpf_rapido = Column(String(20))
    email_rapido = Column(String(255))
    status = Column(String(50), default="confirmada")  # ou "cancelada"
    cancelado_em = Column(DateTime, default=datetime.datetime.utcnow)
    sincronizado = Column(Boolean)

File: eventos-admin/eventos_admin/src/main/java/com/eventosmanager/models/Inscricao.java
package com.eventosmanager.models;

import java.util.UUID;
import java.time.LocalDateTime;

public class Inscricao {
    public UUID id;
    public UUID usuario_id;
    public UUID evento_id;
    public Boolean inscricao_rapida;
    public String nome_rapido;
    public String email_rapido;
    public String cpf_rapido;
    public String status;
    public LocalDateTime inscrito_em;
    public LocalDateTime cancelado_em;
    public Boolean sincronizado;
}
